tests/vcf_test.py

```diff
@@ -43,6 +45,65 @@ def test_header(self, genome_reader):
             assert line
             assert line[0] == "#"
 
+    def test_header(self, blocks, genome_reader):
+        """
+        GIVEN an interable of blocks
+        """
+        vcfgenerator = VCFMaker(genome_reader, {})
+
+        """
+        GIVEN a VCF generator
+        """
+        # generate vcf lines and qc info
+        vcf_lines = [line for line in vcfgenerator.generate_lines(blocks)]
+
+        """
+        WHEN generating a header
+        """
+        lines = tuple(
+            (
+                str(i)
+                for i in vcfgenerator.generate_header("2022-03-25 9:20 AM", "GSAMD-24v3-0-EA_20034606_A2", "GRCh38")
+            )
+        )
+        """
+        THEN it should produce valid header output
+        """
+        header_data = {}
+        for line in lines:
+            print(line)
+            assert line
+            assert line[:2] == "##"
+            field, data = line[2:].split("=", maxsplit=1)
```

Comment:
🤔 I wonder if we want to do our own parsing here, or use `VCFLine` again? Seems like since its a separate library, we could rely on it to do the expected behaviour.